xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml

```diff
@@ -172,16 +175,61 @@ changes: Added @evidence to date
     <text>
         <body>
             <div type="edition">
-            <ab></ab></div>
+            <ab>
+                <lb n="1"/>βριτακυλμα αρωδασιδακ<unclear>ι</unclear><gap reason="lost" quantity="1" unit="character"/><unclear>α</unclear>
+                <lb n="2"/>ηλλιμ αβρι μαχρημελλα
+                <lb n="3"/>αστρακ ιαηου σαθνανω ωπαχ
+                <lb n="4"/>θαλσου σανταρμα χουχι
+                <lb n="5"/>ερηβαθ σαρδοασιρως αχωρ<add place="above">ω</add>
+                <lb n="6"/>ὦ βρώ<supplied reason="omitted">ν</supplied>των καὶ ἀστ<supplied reason="omitted">ρ</supplied>άπτων <name type="divine" key="Iao Ιαω">Ιαω</name>
+                <lb n="7"/>βάλη δῆσων σύ<supplied reason="omitted">ν</supplied>δησω<supplied reason="omitted">ν</supplied> <name key="Βαβύλας Babylas">Βαβυλᾶν</name>
+                <lb n="8"/>τὼν λαχανωπώλην ὥ<supplied reason="omitted">ν</supplied> ἤτηκεν
+                <lb n="9"/>εἱ μιηρὰ μέτρα <name key="Διωνυσία Dionysia">Διωνυσία</name> εἱ κὲ
+                <lb n="10"/><name key="Εἱσυχία Hesykhia">Εἱσυχία</name> ὐκῶ<supplied reason="omitted">ντα</supplied> ἐν γιτωνία Μυγδ
+                <lb n="11" break="no"/>ωνιτῶν: ὡς ἔβαλες τὼ ἅ<supplied reason="omitted">ρ</supplied>μα το
+                <lb n="12" break="no"/>ῦ Φαραῶνος, οὕτος βάλη τὶν δύσ
```

Comment:
Maybe worth putting some name markup around Φαραῶνος here, kind of like how we do with Caesar when it's a title?